code-challenges.js

```diff
@@ -9,67 +9,171 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
+// my pseudocode: we want to create a function which will take in a temperature and create a decision that our temperature is one of three possibilities
+
 var temp1 = 85
 var temp2 = 350
 var temp3 = 212
 
+var randomTemp = Math.floor(Math.random()* 350)
+
+const decideTemperature = (tempStatus) => {
+    if (tempStatus <= 211){
+        return `${tempStatus} is below boiling point.`
+    } else if (tempStatus === 212){
+        return `${tempStatus} is at boiling point.`
+    } else if (tempStatus > 212){
+        return `${tempStatus} is above boiling point.`
+    }
+}
 
+// var temp1
+console.log(decideTemperature(temp1))
 
+// var temp2 
+console.log(decideTemperature(temp2))
 
+// var temp3
+console.log(decideTemperature(temp3))
+
+// random temperature just for kicks
+console.log(decideTemperature(randomTemp))
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
+//my pseudocode: we want a function that will take each value of an array, determine each value within the array meets our conditions, and outputs those values multiplied by 5 in an array
+
 var myForNumbers1 = [3, 7, 0, 6, -9]
 var myForNumbers2 = [2, 5, -8, 20, 32]
 
+const multipliesByFive = (anArray) => {
+    let storageForNewArray = []
+    for (let i=0 ; i < anArray.length ; i++){
+        storageForNewArray.push(anArray[i]*5)
+    }
+    return storageForNewArray 
+}
 
+// expected outcome1 [15, 35, 0, 30, -45]
+console.log(multipliesByFive(myForNumbers1))
 
+// expected outcome2 [10, 25, -40, 100, 160]
+console.log(multipliesByFive(myForNumbers2))
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
+// my pseudocode: we want a function that will take each value of an array and use the built in .map to multiply each value of the array by 5
+
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+const multipliesArrayValuesByFive = (theArrayValue) => {
+   return theArrayValue.map( value =>{
```

Comment:
you can also use implicit returns. (general variable naming conventions leave out articles like `the`
```suggestion
   return arrayValue.map(value => value * 5)
```